src/pages/Login/Login.js

```diff
@@ -1,12 +1,114 @@
-import React from "react";
+import React, { useState } from 'react';
 import './Login.scss';
+import Logo from './logo.png';
+import { useNavigate } from 'react-router-dom';
 
 const Login = () => {
+  const navigate = useNavigate();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const setChangeEmail = (event) => {
+    setEmail(event.target.value);
+  };
+
+  const setChangePassword = (event) => {
+    setPassword(event.target.value);
+  };
+
+  const goToSignUp = () => {
+    navigate('./sign-up');
+  };
+
+  const goToMain = () => {
+    fetch('http://10.58.52.64:8000/users/login', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+      body: JSON.stringify({
+        email: email,
+        password: password,
+      }),
+    })
+      .then((res) => res.json())
+      .then((data) => {
+        if (data.message === 'LOGIN_SUCCESS') {
+          localStorage.setItem('TOKEN', data.token);
+          navigate('/main');
+        } else if (data.message === 'USER_NOT_FOUND') {
+          alert(
+            '가입되지 않은 이메일입니다.\n회원 가입 혹은 이메일을 확인해주세요.',
+          ); // 없는 이메일로 로그인 시도 시 회원 가입 혹은 이메일 확인 권유 알림
+        } else if (data.message === '로그인 정보 불일치!') {
+          alert('비밀번호가 틀렸습니다.'); // 비밀번호가 틀렸음을 알림
+        }
+      });
+  };
+
+  const [active, setActive] = useState(false);
+
+  const passedLogin = () => {
+    return email.includes('@') && password.length >= 10 && email.includes('.')
+      ? setActive(true)
+      : setActive(false);
+  };
```

Comment:
조건이 truthy하면 true, falsy하면 false라고 active state를 업데이트하고 계신데,
state를 만들어서 사용할 필요 없이 조건 자체를 변수로 만들어서 사용하시면 됩니다!
그리고 변수명도 명확하게 지어주시면 좋을 것 같아요.
```suggestion
  const isUserInputValid = email.includes('@') && password.length >= 10 && email.includes('.');
```